### ISSUE TITLE
pulsar source localrun command report error: IllegalArgumentException

### ISSUE THREAD

**Issue by tuteng** [apache/pulsar#4248]
**Describe the bug**
When I run the following command
```
./bin/pulsar-admin source localrun --sourceConfigFile pulsar-io/debezium/mysql/src/main/resources/debezium-mysql-source-config.yaml
```
This error occurred:

```
Exception in thread "main" java.lang.IllegalArgumentException: Source Package is not provided
	at org.apache.pulsar.functions.utils.SourceConfigUtils.validate(SourceConfigUtils.java:261)
	at org.apache.pulsar.functions.runtime.LocalRunner.start(LocalRunner.java:136)
	at org.apache.pulsar.functions.runtime.LocalRunner.main(LocalRunner.java:88)
```

**To Reproduce**
1. Use the latest code

2. Configuration file debezium-mysql-source-config.yaml

```
tenant: "test"
namespace: "test-namespace"
name: "debezium-mysql-source"
topicName: "debezium-mysql-topic"
archive: "connectors/pulsar-io-debezium-mysql-2.4.0-SNAPSHOT.nar"

parallelism: 1

configs:
  ## config for mysql, docker image: debezium/example-mysql:0.8
  ## sourceTask
  task.class: "io.debezium.connector.mysql.MySqlConnectorTask"
  database.hostname: "localhost"
  database.port: "3306"
  database.user: "debezium"
  database.password: "dbz"
  database.server.id: "184054"
  database.server.name: "dbserver1"
  database.whitelist: "inventory"

  ## PULSAR_SERVICE_URL_CONFIG
  pulsar.service.url: "pulsar://127.0.0.1:6650"
  database.history.pulsar.topic: "mysql-history-topic"
  offset.storage.topic: "mysql-offset-topic"
```

3. 
```
docker run -it --rm --name mysql -p 3306:3306 -e MYSQL_ROOT_PASSWORD=debezium -e MYSQL_USER=mysqluser -e MYSQL_PASSWORD=mysqlpw debezium/example-mysql:0.8

cp pulsar-io/debezium/mysql/target/pulsar-io-debezium-mysql-2.4.0-SNAPSHOT.nar ./connectors

./bin/pulsar-admin source localrun --sourceConfigFile debezium-mysql-source-config.yaml

Exception in thread "main" java.lang.IllegalArgumentException: Source Package is not provided
	at org.apache.pulsar.functions.utils.SourceConfigUtils.validate(SourceConfigUtils.java:261)
	at org.apache.pulsar.functions.runtime.LocalRunner.start(LocalRunner.java:136)
	at org.apache.pulsar.functions.runtime.LocalRunner.main(LocalRunner.java:88)
```

**Expected behavior**
localrun success

**Desktop (please complete the following information):**
 - OS: [Mac]

**Additional context**
I feel it has something to do with this pull request https://github.com/apache/pulsar/pull/3969.
@jerrypeng

**Comment by jerrypeng** [comment]
@tuteng I have a fix up.  Are you free to write some integration tests for the local run mode for functions/sources/sinks.  I don't think we have any add this current moment.  I would write them myself, but I am currently busy with some other matters. Thanks in advance!

**Comment by sijie** [comment]
This is fixed by #4278
